.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea
```

Comment:
I'd prefer not to change this file. See [this page](https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally) for info for user-specific ignores.